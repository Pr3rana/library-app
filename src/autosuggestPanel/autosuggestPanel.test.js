import React from 'react';
import renderer from 'react-test-renderer';
import AutoSuggestPanel, {SuggestionRow} from './Autosuggest'

it('renders correctly', () => {
  const tree = renderer
    .create(<AutoSuggestPanel list={[]} onClick={()=>{}}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
it('renders correctly', () => {
    const tree = renderer
      .create(<SuggestionRow title="Test" onClick={()=>{}}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
});
