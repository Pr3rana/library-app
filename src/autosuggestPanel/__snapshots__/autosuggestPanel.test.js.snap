// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  id="autosuggestPanel"
/>
`;

exports[`renders correctly 2`] = `
<div
  className="suggestionRow"
/>
`;
